templates/vas.conf.erb

```diff
@@ -87,7 +87,7 @@
 <% if @vas_conf_preload_nested_memberships != 'UNSET' -%>
  preload-nested-memberships = <%= @vas_conf_preload_nested_memberships %>
 <% end -%>
-<% if !@vas_conf_upm_computerou_attr.empty?-%>
+<% if @vas_conf_upm_computerou_attr != 'UNSET' -%>
```

Comment:
This is dropping support for empty values and breaks backward compatibility by that.